infrastructure/src/main/java/client/AccountsServiceClient.java

```diff
@@ -0,0 +1,77 @@
+package client;
```

Comment:
It makes sense this client will be inside a submodule `client` as part of `accounts-service` module (`:accounts-service:client`). Let's talk about it